### PR TITLE
feat(#1685,1691,1625,1620,1621,1632,1622,1624): new patterns pages

### PR DESCRIPTION
**Pull request by vanessatran-ddi**

### CODE REVIEW

**Comment by lizhuomeng71** [comment]
Spelling for
patterns/simple-form
under
Adding complexity


Expected:
Start by making sure that the content and questions **you** are asking the user **are** as simple as possible.
Actual
Start by making sure that the content and questions your are asking the user is as simple as possible.

**Comment by lizhuomeng71** [comment]
Review Page, Last code Snippet, get console error

Warning: validateDOMNesting(...): <tr> cannot appear as a child of <table>. Add a <tbody>, <thead> or <tfoot> to your code to match the DOM tree generated by the browser.


image

**Comment by lizhuomeng71** [comment]
Result Page, Code Snippet Generated page, not align with example for react

Generated, 
image

Example
image

**Comment by lizhuomeng71** [comment]
Start Page, Angular,

I got this error message

image

**Comment by lizhuomeng71** [comment]
start Page, Generated Page and example not match

Generated

image

Example

image

**Comment by lizhuomeng71** [comment]
Question pages, second title,  Generated Page and example not match

Generated
image
example
image

**Comment by lizhuomeng71** [comment]
Question pages, Multiple Question, Generated Page and example not match, angular

Generated

image


Example
image

**Comment by lizhuomeng71** [comment]
Question pages, Last section, Generated Page and example not match, angular

Generated
image

Example
image

**Comment by lizhuomeng71** [comment]
issue is verified for react and angular, chrome firefox and edge

File: src/routes/root.css [hunk 1]
```diff
@@ -56,6 +56,11 @@ h4.heading {
   margin-bottom: 0;
 }
 
+p.help-text {
```
Comment (chrisolsen) [inline src/routes/root.css:59]: Maybe naming this to `p.small` would make more sense instead of being too specific.

**Review by chrisolsen — APPROVED** [review]

**Pull request merged**